npm/ng-packs/packages/theme-shared/src/lib/directives/ngx-datatable-list.directive.ts

```diff
@@ -39,6 +40,30 @@ export class NgxDatatableListDirective implements OnChanges, OnDestroy, OnInit {
     this.setInitialValues();
   }
 
+  ngDoCheck(): void {
```

Comment:
I believe there is nothing wrong to use **`NgDoCheck`**, it will only check a simple **`if statement`**. Rest of it is similar to what we doing in `ngOnChanges` of **`extensible-table.component.ts`**